src/Northwind.MusicStore.WebUI/Filters/InitializeSimpleMembershipAttribute.cs

```
@@ -60,10 +60,14 @@ public SimpleMembershipInitializer()
                         roleProvider.AddUsersToRoles(new[] { "StoreAdmin" }, new[] { "Administrator" });
                     }
 
-                }
-                catch (Exception ex)
```

A catch-all exception is required.